Application/Api/Education/Certificate/Generator/Repository/BfsHj.php

```diff
@@ -0,0 +1,919 @@
+                ->setContent('<b>{% if(Content.P' . $personId . '.Input.Operation1 is not empty) %}
+                        {{ Content.P' . $personId . '.Input.Operation1 }}
+                    {% else %}
+                        < EINSATZGEBIETE >
+                    {% endif %}</b> (Dauer 
+                     {% if(Content.P' . $personId . '.Input.OperationTime1 is not empty) %}
+                        {{ Content.P' . $personId . '.Input.OperationTime1 }}
+                    {% else %}
+                        X
+                    {% endif %}
+                     Wochen)')
+                ->stylePaddingTop('10px')
+                ->stylePaddingLeft('5px')
+            )
+        );
+
+        $Slice->addSection((new Section())
+            ->addElementColumn((new Element())
+                ->setContent('<b>{% if(Content.P' . $personId . '.Input.Operation2 is not empty) %}
+                        {{ Content.P' . $personId . '.Input.Operation2 }}
+                    {% else %}
+                        < EINSATZGEBIETE >
+                    {% endif %}</b> (Dauer 
+                     {% if(Content.P' . $personId . '.Input.OperationTime2 is not empty) %}
+                        {{ Content.P' . $personId . '.Input.OperationTime2 }}
+                    {% else %}
+                        X
+                    {% endif %}
+                     Wochen)')
+                ->stylePaddingTop('10px')
+                ->stylePaddingLeft('5px')
+                , '60%'
+            )
+            ->addElementColumn((new Element())
+                ->setContent('Dauer gesamt: {{ Content.P' . $personId . '.Input.OperationTime1 + Content.P' . $personId . '.Input.OperationTime2 + Content.P' . $personId . '.Input.OperationTime3 }} Wochen')
+                ->stylePaddingTop('10px')
+                ->styleAlignRight()
+                ->stylePaddingRight('15px')
+                , '40%'
+            )
+        );
+
+        $Slice->addSection((new Section())
+            ->addElementColumn((new Element())
+                ->setContent('<b>{% if(Content.P' . $personId . '.Input.Operation3 is not empty) %}
+                        {{ Content.P' . $personId . '.Input.Operation3 }}
+                    {% else %}
+                        < EINSATZGEBIETE >
+                    {% endif %}</b> (Dauer 
+                     {% if(Content.P' . $personId . '.Input.OperationTime3 is not empty) %}
+                        {{ Content.P' . $personId . '.Input.OperationTime3 }}
+                    {% else %}
+                        X
+                    {% endif %}
+                     Wochen)')
+                ->stylePaddingTop('10px')
+                ->stylePaddingLeft('5px')
+            )
+        );
+
+        return $Slice;
+    }
+
+    /**
+     * @param        $personId
+     * @param string $Height
+     *
+     * @return Slice
+     */
+    private function getDescriptionBsContent($personId, $Height = '195px')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$Height" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)